packages/repository/src/model.ts

```diff
@@ -30,12 +31,30 @@ export interface PropertyDefinition {
   type: PropertyType; // For example, 'string', String, or {}
   id?: boolean | number;
   json?: PropertyForm;
-  jsonSchema?: {[attribute: string]: any};
+  jsonSchema?: PropertyJsonSchemaDefinition;
   store?: PropertyForm;
   itemType?: PropertyType; // type of array
   [attribute: string]: any; // Other attributes
 }
 
+/**
+ * A definition for
+ * {@link https://json-schema.org/specification-links.html#draft-6 | JSON Schema Draft 6}
+ * and LoopBack's custom error message.
+ *
+ * @remarks
+ * This definition is uses
+ * {@link https://json-schema.org/specification-links.html#draft-6 | JSON Schema Draft 6}
+ * and not the
+ * {@link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md | OpenAPI 3}
+ * JSON Schema.
+ *
+ * @public
+ */
+export interface PropertyJsonSchemaDefinition extends JsonSchema {
+  errorMessage?: string | AnyObject;
```

Comment:
What is `errorMessage` for? Can you please add a TSDoc comment to explain?

Reply:
It's the custom AJV error message feature introduced by #4623. I'll add the tsdocs.